### DISCUSSION TITLE
CLASS 'mb-*' not working when added element using jquery .after()

### DISCUSSION THREAD

**Discussion by razorkyle** [twbs/bootstrap#41051] [Q&A]
class 'mb-3' not working after adding new element using jQuery .after(), plea see image below (current display and inspect tab)

CURRENT DISPLAY
![image](https://github.com/user-attachments/assets/9aa6d32c-a4de-4d71-ad27-48d7ce143e2c)

INSPECT TAB
![image](https://github.com/user-attachments/assets/116922a3-5686-4b9b-80d4-0fdfb3962e76)

### Reduced test cases

jQuery v3.7.1

script
![image](https://github.com/user-attachments/assets/6d2b4529-33f6-43d0-b005-6e3eb379e7dc)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

v5.3

**Comment by razorkyle** [comment]
additional information, all the elements are inside 'card-body' class

COMPOSITION
card-group > card > card-body > form > row > col

**Comment by julien-deramond** [comment]
Hi @razorkyle 
I transformed your issue into a discussion, as it's not really a Bootstrap issue since `.mb-2` is just a regular class that simply does `margin-bottom: .5rem !important` (regular CSS).

**Comment by patkon** [comment]
`.card-body` (you're parent container) using `display: flex`. So you can try to using Gap:
https://getbootstrap.com/docs/5.3/utilities/spacing/#gap
